src/Controls/tests/TestCases.HostApp/Issues/Issue28986.xaml

```diff
@@ -0,0 +1,294 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
We want to test Safearea in a few controls

Can you create a few versions of this test?
Just create the new XAML files with an underscore of the control testing

Issue28986_ScrollView
Issue28986_ContentView
Issue28986_Border
Issue28986_ContentPage

This way we can test the behavior of SafeArea for each of these scenarios